modules/acs-central-services-overview.adoc

```diff
@@ -6,15 +6,17 @@
 = Central services
 
 You install Central services on a single cluster.
-These services include three main components, Central, Central DB and Scanner.
+These services include four main components, Central, Central DB, StackRox Scanner, and Scanner V4 (optional).
 
 * *Central*: Central is the {product-title-short} application management interface and services.
 It handles API interactions and user interface ({product-title-short} Portal) access.
 You can use the same Central instance to secure multiple {ocp} or Kubernetes clusters.
 * *Central DB*: Central DB is the database for {product-title-short} and handles all data persistence. It is currently based on PostgreSQL 13.
-* *Scanner*: Scanner is a Red Hat-developed and certified vulnerability scanner for scanning container images. Scanner performs the following functions:
+* *Scanners*: Beginning with release 4.4, {product-title-short} provides two Red Hat-developed and certified vulnerability scanners for scanning container images. Scanner V4, which was developed by consolidating the StackRox and Clair scanners, is the default scanner for language and OS-specific image components. The StackRox Scanner was the default scanner in product-title-short before Scanner V4 was introduced in release 4.4. The StackRox Scanner originates from a fork of the Clair v2 open source scanner. You must continue using this scanner for RHCOS node scanning and for active vulnerability management until that functionality is supported using Scanner V4 in a future release. The scanners perform the following functions:
 
-** It analyzes all image layers and checks for known vulnerabilities from the Common Vulnerabilities and Exposures (CVEs) list.
-** It identifies vulnerabilities in installed packages and dependencies for multiple programming languages.
-In addition to scanning container images, Scanner identifies vulnerabilities in the node's operating system and orchestrators.
+** They analyze all image layers and checks for known vulnerabilities from the Common Vulnerabilities and Exposures (CVEs) list.
```

Comment:
> from the Common Vulnerabilities and Exposures (CVEs) list.

There is no single list like this. We should replace this with our real vuln sources. I will give you that information offline

Reply:
> and checks

and check

Reply:
I put in something that I had elsewhere, please let me know if there are more that I should add or revise the list.

Reply:
Added list you provided in a later comment into the Architecture section.